packages/blockbook/src/controller.ts

```diff
@@ -229,7 +229,7 @@ export class Blockbook extends Controller {
    *
    * @example txid "0xf5ef950d597bea8c980fe5a17faeb70ed7f5030dcf58f077baede5eb0d610cdf"
    */
-  @Example<TxSpecific>({
+  @Example<EthTxSpecific>({
```

Comment:
This is good, we should be showing the corresponding ethereum specific type for the ethereum example. We can add each additional example as coins are added.

```
2. Bitcoin

@example txid "bitcointxid"

@Example<BitcoinTxSpecific>
```
In the appropriate ordering with eth

Reply:
Roger that. Meant to include this.